configuration.go

```diff
@@ -327,6 +327,9 @@ func NewTraefikDefaultPointersConfiguration() *TraefikConfiguration {
 	defaultWeb.Statistics = &types.Statistics{
 		RecentErrors: 10,
 	}
+	defaultWeb.Metrics = &types.Metrics{
+		System: "prometheus",
```

Comment:
I just wanted to avoid exporter as It's a bit confusing with https://prometheus.io/docs/instrumenting/exporters/ but It's fine, Exporter probably better than System anyway